transport/grpc/integration_test.go

```diff
@@ -612,10 +754,10 @@ func createTLSScenario(t *testing.T, clientValidity time.Duration, serverValidit
 			},
 			SerialNumber:          big.NewInt(1),
 			BasicConstraintsValid: true,
-			IsCA:                  true,
-			KeyUsage:              x509.KeyUsageCertSign,
-			NotBefore:             now,
-			NotAfter:              now.Add(10 * time.Minute),
+			IsCA:      true,
+			KeyUsage:  x509.KeyUsageCertSign,
+			NotBefore: now,
+			NotAfter:  now.Add(10 * time.Minute),
```

Comment:
This is weird. Is it my editor config?

Reply:
Try running `make generate SUPPRESSDOCKER=1`. I think that’ll use the containerized `goimports` version and fix this.